releases/2022-01-31-releasenotes.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Improved User Experience and Developer Experience'
+description: "Vendia Share Node Details View gets a New Look, and Smart Contracts Editor improves developer productivity"
+date: '2022-01-31'
+draft: false
+tags:
+- User Experience
+- Developer Experience
+---
+
+# Vendia Share Node Details View gets a New Look
+
+
+## Summary
+
+Our design and engineering teams worked diligently on the node details view redesign. We are excited to share the new interface with you today. You will also see noticeable developer experience improvements on the smart contracts details and the smart contract edit pages. 
+
+
+## Node Details Redesign
+
+Now you don’t have to search hard to find the newly redesigned action buttons - `Entity Explorer`, `GraphQL Explorer`, `File Explorer`, and `Smart Contracts`. They are front and center, ready for use. The details such as the `GraphQL URL` and the `Graph API Key`  have moved below the action buttons and the other resource ARNs that used to be on this page have moved to the `Manage Node' > `Resources` tab.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/96793170/152025230-68abb6a8-2dd3-47fe-b7b3-c89c27177e1e.png" />
+</p>
+<p align="center">Figure 1: Node Details Page</p>
+
+
+## Smart Contracts Editor Enhancements
+
+We added more information to the Smart Contracts details page, including the Vendia resource name of the smart contract. In addition, we've auto-formatted the input query and the output mutation.
+
+
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/96793170/152026225-fab1ad94-f305-4277-803e-844106a7af03.png" />
+</p>
+<p align="center">Figure 2: Smart Contracts details page</p>
+
+Additionally, we are now auto-populating the query args based on the input query when invoking a smart contract. Previously, the `Input Query` Args was empty `{}`. The default value of inferred keys is set to `null`. You will replace `null` with a valid value. 
```

Comment:
```suggestion
Additionally, we now auto-populate the `Input Query Args` input box based on the query provided with the smart contract.
```

Reply:
@db-vendia - Can you confirm I understand the intent here and don't misrepresent?  You and @vikrant-vendia  are closer to this one than I am.  Going for a simple 1 or 2 liner to capture what's changed and what the benefits are to the user.